kafka-ui-react-app/src/components/Topics/List/ActionsCell.tsx

```diff
@@ -70,6 +70,11 @@ const ActionsCell: React.FC<CellContext<Topic, unknown>> = ({ row }) => {
           </>
         }
         danger
+        permission={{
+          resource: ResourceType.TOPIC,
+          action: [Action.MESSAGES_READ, Action.CREATE, Action.DELETE],
```

Comment:
recreating a topic requires VIEW, CREATE and DELETE for topic, messages_read is unrelated at all

Reply:
@Haarolean OK. I replaced Action.MESSAGES_READ to Action.VIEW.